langgraph/checkpoint/memory.py

```diff
@@ -188,6 +239,30 @@ async def alist(self, config: RunnableConfig) -> AsyncIterator[CheckpointTuple]:
             except StopIteration:
                 return
 
+    async def asearch(
+        self,
+        metadata_query: CheckpointMetadata,
```

Comment:
`MemorySaver.asearch()` Does not support `before` and `limit` APIs because `loop.run_in_executor()` doesn't accept named arguments.

Reply:
you can use `functools.partial` for that